### ISSUE TITLE
Expeditor cannot find agent for inspec/kitchen-inspec

### ISSUE THREAD

**Issue by james-stocks** [inspec/kitchen-inspec#251]
Despite having [expeditor config](https://github.com/inspec/kitchen-inspec/blob/master/.expeditor/config.yml) , the expeditor Slack integration reports `I'm sorry, but Expeditor can not find an agent associated with inspec/kitchen-inspec.` when trying to interact with this project.

**Comment by kamaradclimber** [comment]
Apparently kitchen-inspec 1.3.2 has been pushed yesterday to rubygems.org (https://rubygems.org/gems/kitchen-inspec) even if https://github.com/inspec/kitchen-inspec/blob/master/CHANGELOG.md traces that release beginning of Feb 2020.

**Comment by kamaradclimber** [comment]
I may not have the same issue than this one but my symptom is:
> Message: Could not load the 'inspec' verifier from the load path.cannot load such file -- inspec/plugin/v2

since yesterday.

**Comment by james-stocks** [comment]
@kamaradclimber that error relates to the change in kitchen-inspec 1.3.2

What version of InSpec are you using?

**Comment by kamaradclimber** [comment]
1.3.2 (we use latest from rubygems.org).

**Comment by james-stocks** [comment]
I mean the version of InSpec itself. The error you see might be due to using an old InSpec that does not implement the v2 plugin system

**Comment by kamaradclimber** [comment]
Indeed, the resolved version of inspec is 2.1.0 which is far behind the 4.1.x published currently on rubygems. I'll investigate this.
